src/main/java/org/alliancegenome/curation_api/model/entities/PredictedVariantConsequence.java

```diff
@@ -0,0 +1,167 @@
+
+import java.util.List;
+
+import org.alliancegenome.curation_api.constants.LinkMLSchemaConstants;
+import org.alliancegenome.curation_api.interfaces.AGRCurationSchemaVersion;
+import org.alliancegenome.curation_api.model.bridges.BooleanValueBridge;
+import org.alliancegenome.curation_api.model.entities.associations.variantAssociations.CuratedVariantGenomicLocationAssociation;
+import org.alliancegenome.curation_api.model.entities.base.AuditedObject;
+import org.alliancegenome.curation_api.model.entities.ontology.SOTerm;
+import org.alliancegenome.curation_api.view.View;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+import org.hibernate.search.engine.backend.types.Aggregable;
+import org.hibernate.search.engine.backend.types.Projectable;
+import org.hibernate.search.engine.backend.types.Searchable;
+import org.hibernate.search.engine.backend.types.Sortable;
+import org.hibernate.search.mapper.pojo.automaticindexing.ReindexOnUpdate;
+import org.hibernate.search.mapper.pojo.bridge.mapping.annotation.ValueBridgeRef;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.FullTextField;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.GenericField;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.IndexedEmbedded;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.IndexingDependency;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.KeywordField;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import com.fasterxml.jackson.annotation.JsonView;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Index;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@Entity
+@Data
+@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)
+@ToString(callSuper = true)
+@AGRCurationSchemaVersion(min = "2.7.0", max = LinkMLSchemaConstants.LATEST_RELEASE, dependencies = { AuditedObject.class })
+@Schema(name = "PredictedVariantConsequence", description = "POJO representing VEP predicted variant consequence results")
+@Table(indexes = {
+	@Index(name = "predictedvariantconsequence_varianttranscript_index", columnList = "varianttranscript_id"),
+	@Index(name = "predictedvariantconsequence_vepimpact_index", columnList = "vepimpact_id"),
+	@Index(name = "predictedvariantconsequence_polyphenprediction_index", columnList = "polyphenprediction_id"),
+	@Index(name = "predictedvariantconsequence_siftprediction_index", columnList = "siftprediction_id"),
+	@Index(name = "predictedvariantconsequence_createdby_index", columnList = "createdby_id"),
+	@Index(name = "predictedvariantconsequence_updatedby_index", columnList = "updatedby_id")
+})
+public class PredictedVariantConsequence extends AuditedObject {
```

Comment:
Might we need any indexes on the string or int fields?

Reply:
@oblodgett I've added indexes to hgvsProteinNomenclature and hgvsCodingNomenclature.  I wouldn't have thought that indexes would be required on the remaining string/int fields but can add more if you think they'd help.  My general rule for indexes is to add them to foreign keys and anything that's likely to be used as a search field, but maybe that's not the best approach?

Reply:
Well if we were going to pull these to the frontend and need to filter by something then we should add the index, thats is outside of search of course otherwise yes only need them on fk's i guess we can address this later.